frontend/src/App.tsx

```diff
@@ -46,6 +46,9 @@ const App = () => {
               textarea {
                 font-family: inherit;
               }
+              .MuiAutocomplete-popper {
+                z-index: 999999 !important;
```

Comment:
Seems overkill 😁  
Constants have been used so far keeping track of z-indexes `https://github.com/rrebase/knboard/blob/master/frontend/src/const.ts#L129`, which could be prettier, but better than this since. What if something like a toast has to have higher index, more 9s 🤪?